src/containers/Causes/Causes.js

```diff
@@ -24,7 +24,7 @@ class Causes extends Component {
             return <div className="white">loading</div>
           }
           if (error) {
-            return (<div>Error</div>)
+            return <div>Error</div>
```

Comment:
Did you mean to use the error componennt here? or is this just formating @javpet?